spotbugs/etc/messages.xml

```diff
@@ -8615,6 +8622,21 @@ after the call to initLogging, the logger configuration is lost
       ]]>
     </Details>
   </BugPattern>
+  <BugPattern type="SPEC_LIST_REMOVEORADD_CHECK">
+    <ShortDescription>Remove() or add() called during for loop iteration of list. </ShortDescription>
+    <LongDescription>Method {1} is inefficient,because the remove() or add() is not allowed to be called during the for loop iteration of the list.</LongDescription>
```

Comment:
If this is a concurrent modification problem, then "inefficient" is not the right description; this will actually cause errors.